opendj-server-legacy/src/main/java/org/opends/server/backends/pluggable/OnDiskMergeImporter.java

```diff
@@ -34,6 +34,7 @@
 import java.lang.management.MemoryPoolMXBean;
 import java.lang.management.MemoryUsage;
 import java.lang.reflect.Method;
+import java.nio.Buffer;
```

Comment:
Is that import necessary? There are no rerferences in this class to `java.nio.Buffer`
```suggestion
```